docs/en/05_Contributing/13_CSS_Coding_Conventions.md

```diff
@@ -22,14 +22,18 @@ We use [Bootstrap 4](http://v4-alpha.getbootstrap.com/) styles where possible.
 
 We follow the [AirBnB CSS Conventions](https://github.com/airbnb/css)
 and the [BEM](http://getbem.com/) methodology (block-element-modifier).
-File naming and style include ordering is inspired by
-[ITCSS](https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/).
+
+Because we use [Bootstrap 4](http://v4-alpha.getbootstrap.com/) which 
+does not follow [BEM](http://getbem.com/) naming convention there will be 
+a lot of places where class names voilate [BEM](http://getbem.com/). 
+However, please note that they are not a indicator of how to name classes. 
+Use [BEM](http://getbem.com/) conventions where possible.
```

Comment:
I'm not sure I agree with all the `BEM` keywords being links, you end up with docs dotted with links repeatedly.
Maybe link one per paragraph should be enough.